packages/sdk/governance/src/proposals.ts

```diff
@@ -1,31 +1,45 @@
-import { Address, isHexString, trimLeading0x } from '@celo/utils/lib/address'
```

Comment:
is there a `governance/proxy-utils` missing here?

Reply:
I've added a Proxy file in the CK, because the proxies are things used in a few parts
Maybe you are referring to that